src/ioctls/vcpu.rs

```diff
@@ -1212,12 +1212,23 @@ impl VcpuFd {
     /// let vm = kvm.create_vm().unwrap();
     /// let vcpu = vm.create_vcpu(0).unwrap();
     ///
-    /// // KVM_GET_REG_LIST demands that the vcpus be initalized.
-    /// let mut kvi: kvm_bindings::kvm_vcpu_init = kvm_bindings::kvm_vcpu_init::default();
-    /// vm.get_preferred_target(&mut kvi).unwrap();
-    /// vcpu.vcpu_init(&kvi).expect("Cannot initialize vcpu");
+    /// // KVM_GET_REG_LIST on Aarch64 demands that the vcpus be initalized.
```

Comment:
nit: missing the 3rd 'i' in initialized

Reply:
Nice catch :)

Reply:
Still missing :D

Reply:
Sorry, somehow missed, thanks for pointing out! :)